### PR TITLE
feat: Support real time add missing template in cache on template message render

### PR DESCRIPTION
**Pull request by liamcho**
Fixes: [<TICKET_ID>](https://sendbird.atlassian.net/browse/<TICKET_ID>)

### Changelogs
-

#### Loading template design
Screenshot 2024-02-25 at 12 11 16 AM

#### Result
Screenshot 2024-02-25 at 12 13 23 AM
Screenshot 2024-02-25 at 12 13 39 AM

#### Fallback template message design 
Screenshot 2024-02-24 at 10 59 54 PM

##### When there is no message.message 
Screenshot 2024-02-25 at 12 11 35 AM
Screenshot 2024-02-25 at 12 11 52 AM

##### When there is message.message
Screenshot 2024-02-25 at 12 12 27 AM
Screenshot 2024-02-25 at 12 12 54 AM

### CODE REVIEW

File: src/modules/App/stories/index.stories.js [hunk 1]
```diff
@@ -269,16 +269,37 @@ export const Korean = () => fitPageSize(
   />
 );
 
-const customApiHost = 'https://api-preprod.sendbird.com';
-const customWebSocketHost = 'wss://ws-preprod.sendbird.com';
+const PersonalTestConfig = {
```
Comment (liamcho) [inline src/modules/App/stories/index.stories.js:272]: 테스트용 임시 코드입니다. 머지전에 제거할 예정입니다.

File: src/modules/GroupChannel/context/hooks/useMessageActions.ts [hunk 2]
```diff
@@ -46,52 +46,15 @@ const pass = <T>(value: T) => value;
...
-      return params;
-    },
+    // FIXME: Put this back to pass before merging.
+    // onBeforeSendUserMessage = (params) => {
+    //   params.extendedMessagePayload = {
+    //     // template: PersonalTemplatesDataForTesting[0],
+    //     template: { key: 't0' },
+    //   };
+    //   return params;
+    // },
```
Comment (liamcho) [inline src/modules/GroupChannel/context/hooks/useMessageActions.ts:56]: 위랑 같습니다

File: src/lib/Sendbird.tsx [hunk 3]
```diff
@@ -201,8 +212,153 @@ const SendbirdSDK = ({
...
 
+  const messageTemplatesInfo: MessageTemplatesInfo | undefined = appInfoStore?.messageTemplatesInfo;
+  const {
+    INITIALIZE_MESSAGE_TEMPLATES_INFO,
+    UPSERT_MESSAGE_TEMPLATE,
+    UPSERT_WAITING_TEMPLATE_KEY,
+    MARK_ERROR_WAITING_TEMPLATE_KEY,
+  } = APP_INFO_ACTIONS;
+
   useTheme(colorSet);
```
Comment (HoonBaek) [inline src/lib/Sendbird.tsx:224]: Sendbird 파일 안에 message template과 관련된 로직들이 많이 들어가서, 분리가 되면 좋을 것 같은데 어떻게 생각하시나요?

Comment (liamcho) [reply src/lib/Sendbird.tsx:206]: https://github.com/sendbird/sendbird-uikit-react/pull/987/commits/28be8aab468474a07a2397384d513932df999e3d
반영했습니다~

**Review by HoonBaek — APPROVED** [review]

**Pull request merged**